Pets.DAL/Interfaces/IFoodRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Pets.DAL.Entities;
+
+namespace Pets.DAL.Interfaces
+{
+    public interface IFoodRepository: IGenericRepository<FoodEntity>
```

Comment:
why it is so empty?

Reply:
There is all realisation in base interface

Reply:
it isn't reason make an empty interface or class

Reply:
I use this empty interface for DI